resources/lib/qobuz/node/user.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, parent=None, parameters=None, data=None):
         self.is_folder = True
         self.data = self.fetch()
 
-    def get_label(self, default=None):
+    def get_label(self):
```

Comment:
The base class signature for **get_label** is **def get_label(self, default=None):**
I think that codacity will not be happy with that change.